models/holidays.js

```diff
@@ -0,0 +1,56 @@
+var mongoose = require('mongoose');
+
+// Article Schema
+var holiSchema = mongoose.Schema({
+  username:{
+    type: String,
+    required: true
+  },
+  jan: {
+		type: Number
+	},
+  feb: {
+   type: Number
+ },
+ mar: {
+   type: Number
+ },
+ april: {
+   type: Number
+ },
+ may: {
+   type: Number
+ },
+ june: {
+   type: Number
+ },
+ july: {
+   type: Number
+ },
+ aug: {
+   type: Number
+ },
+ sept: {
+   type: Number
+ },
+ oct: {
+   type: Number
+ },
+ nov: {
+   type: Number
+ },
+ dec: {
+   type: Number
+ }
+});
+
+var Holiday = module.exports = mongoose.model('Holiday', appSchema);
+
+module.exports.getUserByUsername = function(username, callback){
+	var query = {username: username};
+	User.findOne(query, callback);
```

Comment:
There is no model 'User' here, so function will give wrong callbacks. It should be holischema